css/style.css

```diff
@@ -98,6 +226,7 @@ body {
 .playlist_links a {
 	flex: 50%;
 	text-align: center;
+	text-decoration: none;
```

Comment:
I decided that the links to the playlist pages don't need to look like links because they are more like buttons to the playlists which is why I removed the underline